src/java/arjdbc/jdbc/ConnectionFactory.java

```diff
@@ -49,84 +43,3 @@
     Connection newConnection() throws SQLException;
 
 }
-
-class DataSourceConnectionFactoryImpl implements ConnectionFactory {
```

Comment:
don't recall but these are no longer used than?
... do we still support `data_source:` option along side with `jndi:`

Reply:
yeah..that one I tripped over...there's a class in its own file, DataSourceConnectionFactory that is actually used, this one is stale. So I moved the other two out to separate files and deleted that one.